Muslim-Software-Developers/Musdev-fe
# Auth

# What's in this PR?

## Checklist

- [ ] Specs (if applicable)
- [x] Visual (Screenshot / GIF / mp4) of new feature (if applicable)

## PR Description



## How to Test
- Go to the Login Page by clicking on the Login button in the Navbar
- Enter email and password, then click on Login Button
- You should get redirected to the Dashboard

1. Do this
2. Then this
3. ...
4. Magic! 🪄
